examples/vertx-opentracing/README.md

```diff
@@ -0,0 +1,153 @@
+# OpenTracing - Microservices with vert.x
+
+This application provides a simple order management system. When a user places an order, the Order Manager communicates with:
+
+- Account Manager to verify the account id is known
+- Inventory Manager to check there is enough stock
+- Once the order is confirmed and acknowledged to the user, it will be stored in the Order Log and the Inventory Manager
+will be updated.
+
+NOTE: Once the end user response has been returned, to confirm the order, the notification to the Order Log and Inventory
+Manager is performed via publishing a message on a shared destination.
+
+The other operation that can be performed by an end user is to list the current orders associated with an account id.
+
+This example features a pub-sub architecture via an event bus, using [Eclipse Vert.x](http://vertx.io). Each `Verticle` is a
+microservice and is deployed into a different service under OpenShift. Hawkular APM acts here as an OpenTracing provider and is
+able to capture data from all services, displaying this data on its UI, like the screenshots below:
+
+![Distributed tracing - Aggregated view](screenshot-1.png?raw=true "Distributed tracing - Aggregated view")
+
+
+![Distributed tracing - Instance view](screenshot-2.png?raw=true "Distributed tracing - Instance view")
+
```

Comment:
It might be good to have a heading here, something like "Running on a local OpenShift cluster" - and then have the two options under that heading, with the "Running outside OpenShift .." at the same level as this new header.